content/articles/php-sessions-and-cookies-to-create-a-secure-remember-me-section-in-a-login-script/Index.md

```diff
@@ -0,0 +1,356 @@
+### Introduction
+In the Login Script, the 'Remember Me' functionality allows users who have logged in to keep their status. A user's logged-in status is serialized in PHP `sessions`, `cookies`, or other similar storage when they select the `Remember Me` option.
+In this edge, you will learn how to create a persistent PHP web application.
+#### Table of contents
+- [Introduction](#introduction)
+- [Preserve logged in state](#create-cookies-to-preserve-the-logged-in-state)
+- [validate remembered login](#validate-remembered-login-with-php-session-and-cookies)
+- [Clear remembered login](#clear-remembered-login-with-session-and-cookies-on-logout)
+- [Controller classes](#auth-and-dbcontroller-classes)
+- [Conclusion](#conclusion)
+- [check the code snippets at github](https://github.com/Juskam/secure-remember-me-for-login-using-php-session-and-cookies)
+Make sure to look at the source code on GitHub so you know what files to create.

```

Comment:
Do not force readers to visit your Github source code . Cover everything on this tutorial. Github is just a reference for interested readers.